phylogenetic/outbreak-specific/config.yaml

```diff
@@ -6,35 +6,72 @@
 
 id_column: "accession"
 
+inputs:
+  - name: local_ingest
+    metadata: ../ingest/results/metadata.tsv
+    sequences: ../ingest/results/sequences.fasta
+
 builds:
+  - drc-2025
   - west-africa-2014
 
 build_params:
+  drc-2025:
+    ancestral:
+      inference: joint
+
+    align:
+      reference: &Yambuku_Mayinga_Reference outbreak-specific/drc-2025/NC_002549.1.gb
```

Comment:
Re: ca77a27

> This duplicates the "reference" path but YAML anchors make this ok

I don't think this is a good pattern for users of `--configfile`, since it means they would have to override multiple config values instead of just one. If a user overrides `align.reference` but not `translate.reference`, I imagine the workflow will run fine but produce unexpected/inaccurate results. We could add error handling to warn/error if `align.reference` and `translate.reference` are the same file, but if there is an expectation that they are the same, then it should just be a single config value in the first place.

Reply:
Fair enough. I've dropped the duplication & anchors from this work.

When we move to small-multiple (expanded, pre-processed) configs we will want this duplication and so we'll have to consider whether to duplicate it in the user-facing config or as part of the pre-processing code.